modules/is_master/manifests/params.pp

```diff
@@ -22,47 +22,8 @@
   $product_version = '5.8.0'
   $service_name = 'wso2is'
 
-  $template_list = [
-    'repository/conf/carbon.xml',
-    'repository/conf/user-mgt.xml',
-    'repository/conf/datasources/master-datasources.xml',
-    'repository/conf/axis2/axis2.xml',
-    'repository/conf/identity/identity.xml',
-  ]
-
-  # carbon.xml configs
-  $ports_offset = 0
-  /*
-     Host name or IP address of the machine hosting this server
-     e.g. www.wso2.org, 192.168.1.10
-     This is will become part of the End Point Reference of the
-     services deployed on this server instance.
-  */
-  $hostname = 'localhost'
-  $mgt_hostname = 'localhost'
-
-  $security_keystore_location = '${carbon.home}/repository/resources/security/wso2carbon.jks'
-  $security_keystore_type = 'JKS'
-  $security_keystore_password = 'wso2carbon'
-  $security_keystore_key_alias = 'wso2carbon'
-  $security_keystore_key_password = 'wso2carbon'
-
-  $security_internal_keystore_location = '${carbon.home}/repository/resources/security/wso2carbon.jks'
-  $security_internal_keystore_type = 'JKS'
-  $security_internal_keystore_password = 'wso2carbon'
-  $security_internal_keystore_key_alias = 'wso2carbon'
-  $security_internal_keystore_key_password = 'wso2carbon'
-
-  $security_trust_store_location = '${carbon.home}/repository/resources/security/client-truststore.jks'
-  $security_trust_store_type = 'JKS'
-  $security_trust_store_password = 'wso2carbon'
-
-  $clustering_enabled = 'false'
-  $clustering_membership_scheme = 'multicast'
-
   # Directories
   $products_dir = "/usr/local/wso2"
-  $puppet_env = "/etc/puppet/code/environments/production"
```

Comment:
Why did we remove this?

Reply:
The `puppet_env` parameter is not being used